### PR TITLE
Add 15-Password_Validation.

### PR DESCRIPTION
**Pull request by sagar627**
Resolves Issue #15

### Description
Simple Website which checks the Password Strength.

### How to run
Here is the repl.it code (https://repl.it/repls/HonestRepentantCodes#index.html).

### Checklist
- [x] Code compiles correctly.
- [x] Changes are tested properly.
- [x] Added the README / documentation, if necessary.
- [x] Followed the Contributing guidelines.
- [x] Resolved merge conflicts, if any.

### CODE REVIEW

**Comment by krithikvaidya** [comment]
@sagar627 please remove instruction text in the PR description and replace it with the relevant details

**Comment by gaurang2001** [comment]
@sagar627 Hey, sorry for the delay. Could you add a section in the HTML page that tells the user the criteria of classification of a password into weak, strong, etc?

Everything else looks fine :+1:

**Comment by sagar627** [comment]
Added guidelines for Passwords. Check out (https://repl.it/repls/HonestRepentantCodes#index.html)

**Comment by gaurang2001** [comment]
@sagar627 your code on the site you've sent works but you haven't added a commit here xD. Please do that.
Additionally, there needs to be at least one special character according to your guidelines but your code in JS doesn't check for it.

File: Development/15-Password_Validation/app.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+password.addEventListener("keyup", checkPassword,false);
+
+function checkPassword() {
+    console.log(password.value);
+    if (password.value == null || password.value == ''){ 
+        PasswordCheck.innerText ="Enter Password";
+        dot.style.backgroundColor = '#f6f6f6';
+    }
+    else {
+        if (password.value.length < 7 ){
```
Comment (gaurang2001) [inline Development/15-Password_Validation/app.js]: change it to 8, according to guidelines you've specified.

**Comment by sagar627** [comment]
@gaurang2001 Updated all code. If any further issue then let me know.

**Comment by sagar627** [comment]
@gaurang2001 Update all file.

**Comment by gaurang2001** [comment]
@sagar627 You added your js code to the css file by mistake. Please change. Will merge post that.

**Comment by sagar627** [comment]
This is my first time using GitHub, thank you @gaurang2001 for helping me.

**Review by gaurang2001 — APPROVED** [review]

**Comment by gaurang2001** [comment]
@krithikvaidya please merge

**Pull request merged**